packages/visualizations/src/components/MapPoi/MapRender.svelte

```diff
@@ -53,6 +55,8 @@
             bounds: bbox as LngLatBoundsLike,
             ...getCenterZoomOptions({ zoom, center }),
             transformRequest,
+            minZoom,
+            maxZoom,
```

Comment:
Currently these props are not reactive after the map has been mounted, I think we need functions to update min and max zoom.